pmlc/tools/pmlc-jit/pmlc-jit.cc

```diff
@@ -96,6 +97,7 @@ int main(int argc, char **argv) {
   llvm::InitializeNativeTarget();
   llvm::InitializeNativeTargetAsmPrinter();
   mlir::initializeLLVMPasses();
+  pmlc::rt::Runtime::initRegisteredRuntimes();
```

Comment:
Would be nice if we had `pmlc::InitializeRuntimes()` so that we are a little more consistent with LLVM.

Reply:
Done